src/symbolic.jl

```diff
@@ -93,7 +93,7 @@ function simplify(ex::Expr)
         return ex
     end
     if all(isnumber, ex.args[2:end]) && length(ex.args) > 1
-        return eval(@__MODULE__, ex)
+        return (@static (VERSION < v"1.0-") ? eval : Core.eval)(@__MODULE__, ex)
```

Comment:
This one is https://github.com/JuliaLang/julia/pull/27098. Would be good to use `VERSION < v"0.7.0-DEV.5149"` to avoid a depwarn on 0.7.

Reply:
okay, done now